sdk/rust/oak_app_build/src/main.rs

```diff
@@ -14,15 +14,17 @@
 // limitations under the License.
```

Comment:
Should it be called `oak_app_builder` or `oak_app_build`?

Reply:
I like oak_app_build better, but I don't have a strong preference. I also thought that this is more a linker, so maybe it should be called oak_app_link(er)? Anyways, we can easily change the name in the future if we like something else.